server/fleet/apple_mdm.go

```diff
@@ -206,9 +206,9 @@ type MDMAppleConfigProfile struct {
 	// Checksum is an MD5 hash of the Mobileconfig bytes
 	Checksum []byte `db:"checksum" json:"checksum,omitempty"`
 	// Labels are the associated labels for this profile
-	Labels    []ConfigurationProfileLabel `db:"labels" json:"labels,omitempty"`
-	CreatedAt time.Time                   `db:"created_at" json:"created_at"`
-	UpdatedAt time.Time                   `db:"updated_at" json:"updated_at"`
+	Labels     []ConfigurationProfileLabel `db:"labels" json:"labels,omitempty"`
+	CreatedAt  time.Time                   `db:"created_at" json:"created_at"`
+	UploadedAt time.Time                   `db:"uploaded_at" json:"updated_at"` // TODO: should we update the JSON rendering?
```

Comment:
I don't think we should, that would be an API breaking change (documented e.g. here: https://fleetdm.com/docs/rest-api/rest-api#default-response61)

Reply:
:+1: sounds good, I'll remove the TODO and add that note.